compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3878,13 +3878,11 @@ object Parsers {
         val ident = termIdent()
         var name = ident.name.asTermName
         val paramss =
-          if in.featureEnabled(Feature.clauseInterleaving) then
-            // If you are making interleaving stable manually, please refer to the PR introducing it instead, section "How to make non-experimental"
+          if Feature.clauseInterleavingEnabled(using in.languageImportContext) then
```

Comment:
This looks out of line with other code that's version dependent, Why the `using in.languageImportContext`?
That's done nowhere else in the parser. Why not just the original test?

Reply:
There's a doc comment further up that needs to be updated. 
```

     * if clauseInterleaving is enabled:
     *  DefSig  ::=  id [DefParamClauses] [DefImplicitClause]
     * 
  ```
As far as Parser is concerned, clauseInterleaving _is_ now enabled, so the grammar should reflect this.